test/service/test_logging.py

```diff
@@ -0,0 +1,65 @@
+import json
+import re
+import requests
+
+from service import WebServiceTestCase
+
+from azul.chalice import log
+from azul.logging import configure_test_logging
+
+
+def setUpModule():
+    configure_test_logging()
+
+
+class TestRequestResponse(WebServiceTestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls._setup_indices()
+
+    @classmethod
+    def tearDownClass(cls):
+        cls._teardown_indices()
+        super().tearDownClass()
+
+    def _get_request_logs(self, level, endpoint, params={}):
+        with self.assertLogs(logger=log, level=level) as logs:
+            url = self.base_url + endpoint
+            requests.get(url, params=params)
+            return logs.output
+
+    @classmethod
+    def _valid_json(cls, json_string):
+        try:
+            json.loads(json_string)
+        except ValueError:
+            return False
+        return True
+
+    def test_request_logs_debug_off(self):
+        logs = self._get_request_logs(level='INFO', endpoint='/health')
+        log_started_count = sum([1 for msg in logs if re.search(r'^INFO.*Received GET request to \'/health\'', msg)])
+        log_ended_count = sum([1 for msg in logs if re.search(r'^INFO.*Returning 200 response.*set AZUL_DEBUG', msg)])
+        self.assertEqual(1, log_started_count)
+        self.assertEqual(1, log_ended_count)
+
+    def test_request_logs_debug_on(self):
+        logs = self._get_request_logs(level='DEBUG', endpoint='/health')
+        log_started_count = 0
+        log_ended_count = 0
+        for msg in logs:
+            if re.search(r'^INFO.*Received GET request to \'/health\'', msg):
+                log_started_count += 1
+            if re.search(r'^DEBUG.*Returning 200 response.*See next line', msg):
+                log_ended_count += 1
+                match = re.search(r'See next line.*\n({.*})$', msg)
+                self.assertIsNotNone(match)
+                json_string = match.group(1)
+                self.assertGreater(len(json_string), 2)
+                self.assertTrue(self._valid_json(json_string))
```

Comment:
Can you assert the literal content of the response, not just that it is valid JSON. You don't have to use the /health endpoint. And also, if you stick with health, I want you to run this test with network cable pulled and wifi off. I have the feeling it is hitting actual cloud resources.

Reply:
`/health` returns a 500 with no internet access. Changing tests to use `/health/basic` which returns a simple `{"up": true}`